build.sbt

```diff
@@ -1,14 +1,13 @@
 name := "spark-dicom"
 
-version := "0.1"
-
 inThisBuild(
   List(
     scalaVersion := "2.12.15",
     semanticdbEnabled := true,
     semanticdbVersion := scalafixSemanticdb.revision, // only required for Scala 2.x
     scalacOptions += "-Ywarn-unused-import", // Scala 2.x only, required by `RemoveUnused`
-    scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0"
+    scalafixDependencies += "com.github.liancheng" %% "organize-imports" % "0.6.0",
+    versionScheme := Some("early-semver"),
```

Comment:
Could you add a comment to describe rationale?

Reply:
I followed the recommendation made here: https://www.scala-lang.org/blog/2021/02/16/preventing-version-conflicts-with-versionscheme.html.

Or do you mean a comment in the code to describe it?

Reply:
If it's a common practice found in many scala project, I guess it's not important